taipy/core/_orchestrator/_orchestrator_factory.py

```diff
@@ -87,7 +87,13 @@ def __build_standalone_job_dispatcher(cls, force_restart=False):
     def __build_development_job_dispatcher(cls):
         if isinstance(cls._dispatcher, _StandaloneJobDispatcher):
             cls._dispatcher.stop()
-        cls._dispatcher = _DevelopmentJobDispatcher(typing.cast(_AbstractOrchestrator, cls._orchestrator))
+
+        if util.find_spec(cls._TAIPY_ENTERPRISE_MODULE) is not None:
```

Comment:
I guess you ll want to use the _using_enterprise function from common package.

Reply:
ah yes, ohh I merged the other PR before this one 🤣